source/frontend/src/features/properties/map/project/ProjectContainer.tsx

```diff
@@ -107,6 +111,11 @@ const ProjectContainer: React.FunctionComponent<
 
   useEffect(() => setProjectLoading(loadingProject), [loadingProject, setProjectLoading]);
 
+  const onSuccess = () => {
+    fetchProject();
```

Comment:
does this need to be an await so that the state is only updated after the fetch is complete?